### PR TITLE
Create reforgeBidAdapter.js

### PR DESCRIPTION
**Pull request by Shivani77777**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'reforge',
                       params: {
                           placementId: 'mediumrectangle-demo',
                           categories: ['Games'] // optional: categories of page
                       }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ shivani.garg@svgmedia.in] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by robertrmartinez** [comment]
@Shivani77777 A corresponding test file and .md file is required.

Please add unit tests and a markdown which contain valid test params which return test ads.

File: modules/reforgeBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import * as utils from '../src/utils.js';
+import {config} from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'reforge';
+const ENDPOINT_URL = 'http://use.reforge.in/bid?rtb_seat_id=0011&secret_key=wzP8eKAVkc&type=prebid&appid=40';
```
Comment (FilipStamenkovic) [inline modules/reforgeBidAdapter.js:7]: You should use `https` endpoint.

File: modules/reforgeBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+   * @param {ServerResponse} serverResponse A successful response from the server.
+   * @param bidRequest
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function (serverResponse, bidRequest) {
+    const bidResponses = [];
+    const response = serverResponse.body;
+
+    response.forEach(function(bid) {
+      bid.ttl = config.getConfig('_bidderTimeout');
```
Comment (FilipStamenkovic) [inline modules/reforgeBidAdapter.js:75]: Are you sure this is what you want, `bidderTimeout` is in milliseconds, while `ttl` is in seconds?

**Review by FilipStamenkovic — CHANGES_REQUESTED** [review]
@Shivani77777 @robertrmartinez  
This new bidder is exactly the same as `ablida` bidder we already have in pbjs. Only `code` and `endpoint` are different.    
I think this bidder should be defined as alias?

**Pull request closed without merging**